src/BundleAnalyzerPlugin.js

```diff
@@ -55,14 +55,19 @@ class BundleAnalyzerPlugin {
 
       if (actions.length) {
         // Making analyzer logs to be after all webpack logs in the console
-        setImmediate(() => {
-          actions.forEach(action => action());
+        setImmediate(async () => {
+          try {
+            for (let i = 0, len = actions.length; i < len; ++i) await actions[i]();
```

Comment:
I don't think we need to pre-optimise the loop here. AFAIK V8's optimising compiler does this for us automatically.

Reply:
You mean write it as `for (let i = 0; i < actions.length; ...` instead?

Reply:
Yes, IMO `for(let i = 0; i < actions.length; i++)` more readable than `for(let i = 0, len = actions.length; i < len; ++i )`, plus it wouldn't make a huge difference in terms of perf.

Reply:
Sure, no problem.